samcli/commands/check/lib/ask_question.py

```diff
@@ -0,0 +1,36 @@
+"""
+A utility method used to ask the user a question. An int is returned only
+if it is within the specified range.
+"""
+
+import click
+
+
+def ask(question: str, min_val: float = 1, max_val: float = float("inf")) -> int:
```

Comment:
nit: Regarding to `int`->`float` discussion, you may think about adding overloaded versions of `ask` which will have different types for the min and max values.